mevislab.github.io/content/tutorials/basicmechanisms/macromodules/soviewportregion.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Example 6: Creating Multi-View Layouts Using SoViewportRegion"
+date: 2025-04-22
+status: "OK"
+draft: false
+weight: 460
+tags: ["Beginner", "Tutorial", "SoViewportRegion", "Layout", "Multi-View"]
+menu: 
+  main:
+    identifier: "soviewportregion"
+    title: "Creating Multi-View Layouts Using SoViewportRegion"
+    weight: 460
+    parent: "basicmechanisms"
+---
+# Example 6: Creating Multi-View Layouts Using SoViewportRegion
+
+## Introduction
+In this guide, we will show how to use the `SoViewportRegion` module to create custom layouts within the `SoRenderArea` module. This allows you to display multiple views or slices in a single window.
+
+We will demonstrate how to: 
+* Divide the render area into multiple regions. 
+* Assign different content to each region. 
+* Use alternative methods, such as SoView2D, when applicable.
+
+
+## Prepare your network
+
+### Displaying three images in one panel
+Add an `ImageLoad` module to your workspace and select 3D image like *./MeVisLab/Resources/DemoData/MRI_Head.tif* from MeVisLab demo data directory. Connect an `OrthoReformat3` module and add three `View2D` modules.
+
+![Image Display Setup](/images/tutorials/basicmechanics/E6_1.png "Image Display Setup")
+
+Opening the three `View2D` module panels now show the data in 3 orthogonal views. The module `OrthoReformat3` transforms the input image (by rotating and/or flipping) into the three main views commonly used.
+
+![3 Views in 3 Viewers](/images/tutorials/basicmechanics/E6_2.png "3 Views in 3 Viewers")
+
+The module `SoViewportRegion` divides the render window into multiple areas, allowing different views or slices to be shown in the same window. It's useful in medical applications, like displaying MRI or CT images from different angles (axial, sagittal, coronal) at once, making data analysis easier and faster.
+
+Add three `SoViewportRegion` modules and connect each one to a `View2D` module. To display the hidden outputs of the `View2D` module, press {{< keyboard "SPACE" >}} and connect the output to the input of `SoViewportRegion`, as shown below. 
+
+![Connect SoViewportRegion with View2D](/images/tutorials/basicmechanics/E6_3.png "Connect SoViewportRegion with View2D")
+
+Add a `SoRenderArea` for your final result to the network and connect all three `SoViewportRegion` modules to it.
+
+The result is, that all of your viewers are initially above each other in the bottom right corner.
```

Comment:
The result is that (no comma)
are initially above each other -> are initially displayed on top of each other